kratos/tests/cpp_tests/geometries/test_coupling_geometry.cpp

```diff
@@ -51,8 +60,8 @@ namespace Kratos {
                 );
         }
 
-        /// Tests
-        KRATOS_TEST_CASE_IN_SUITE(CouplingNode, KratosCoreGeometriesFastSuite) {
+        /// Test with PointType = Node
```

Comment:
it would be better if you have an aux-fct that does the test and that is templated with `PointType`
This way you wouldn't have to copy-paste the entire test
(Tests btw take a large part of the compilation time already now)

Reply:
Shouldn't tests take that much extra time, because otherwise big parts of the code would be neglected by the compiler?

Are sure that this would reduce the compilation time? I thought that within templated functions the compiler needs to compile each variation.

Anyways, I see your issue, thanks. With both tests, we want to test different functions. Thus, one is reduced significantly.